modal/shared/protocol.py

```diff
@@ -77,17 +46,12 @@ class Params(BaseModel):
     skip_special_tokens: bool = True
 
 
-class RunnerConfiguration(BaseModel):
-    container: ContainerType
-
-
 class CompletionPayload(BaseModel):
     id: str
     prompt: str
     stream: bool = False
     params: Params
     model: str
-    runner: RunnerConfiguration | None = None
```

Comment:
opted to get rid of this complexity so theres less going on.  just a simple 1-to-1 mapping for model->container

Reply:
looks like -web repo does not use this field so we good

https://github.com/OpenRouterTeam/openrouter-web/blob/05061d4f8bb3ae41b207a6784c78c9da7401a05c/utils/adapters/modal.ts#L81-L101